src/NoteDisplay.cpp

```diff
@@ -425,29 +716,25 @@ struct StripBuffer
 	int Free() const { return size - Used(); }
 };
 
-void NoteDisplay::DrawHoldPart( vector<Sprite*> &vpSpr, int iCol, int fYStep, float fPercentFadeToFail, float fColorScale, bool bGlow,
-				float fDrawDistanceAfterTargetsPixels, float fDrawDistanceBeforeTargetsPixels, float fFadeInPercentOfDrawFar,
-				float fOverlappedTime,
-				float fYTop, float fYBottom,
-				float fYStartPos, float fYEndPos,
-				bool bWrapping, bool bAnchorToTop, bool bFlipTextureVertically )
+void NoteDisplay::DrawHoldPart(vector<Sprite*> &vpSpr,
```

Comment:
I agree, I'll look at the code some more to condense the parameter list after I do some testing on other aspects.  NoteField and NoteDisplay in general have huge parameter lists, often the same for multiple functions, and I think that make tracking what is passed more difficult.